download-ecosystem-tools.md

```diff
@@ -68,18 +68,3 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
 > 下载链接中的 `{version}` 为 Dumpling 的版本号。例如，`v4.0.2` 版本的下载链接为 `https://download.pingcap.org/tidb-toolkit-v4.0.2-linux-amd64.tar.gz`。可以通过 [Dumpling Release](https://github.com/pingcap/dumpling/releases) 查看当前已发布版本。
 > Dumpling 已支持 arm64 linux，将下载链接中的 amd64 替换为 arm64，即表示 arm64 版 Dumpling。
 
-## Syncer，Loader 和 Mydumper
```

Comment:
@IANTHEREAL hi please help confirm if we can delete this section. (because it gives instructions for 3 deprecated tools.)

Reply:
need to add that the download links of sync-diff-inspector and ticdc

Reply:
Updated in https://github.com/pingcap/docs-cn/pull/5213/commits/eabbf58fc310a624ac3b2bf32038609244378a6a